cmd/query/app/flags.go

```diff
@@ -23,21 +23,26 @@ import (
 	"net/http"
 	"net/textproto"
 	"strings"
+	"time"
 
 	"github.com/spf13/viper"
 	"go.uber.org/zap"
 
+	"github.com/jaegertracing/jaeger/cmd/query/app/querysvc"
+	"github.com/jaegertracing/jaeger/model/adjuster"
 	"github.com/jaegertracing/jaeger/pkg/config"
 	"github.com/jaegertracing/jaeger/ports"
+	"github.com/jaegertracing/jaeger/storage"
 )
 
 const (
-	queryPort              = "query.port"
-	queryBasePath          = "query.base-path"
-	queryStaticFiles       = "query.static-files"
-	queryUIConfig          = "query.ui-config"
-	queryTokenPropagation  = "query.bearer-token-propagation"
-	queryAdditionalHeaders = "query.additional-headers"
+	queryPort               = "query.port"
+	queryBasePath           = "query.base-path"
+	queryStaticFiles        = "query.static-files"
+	queryUIConfig           = "query.ui-config"
+	queryTokenPropagation   = "query.bearer-token-propagation"
+	queryAdditionalHeaders  = "query.additional-headers"
+	queryMaxClockSkewAdjust = "query.max-clock-skew-adjust"
```

Comment:
nit: "adjust" is not a noun
```suggestion
	queryMaxClockSkewAdjust = "query.max-clock-skew-adjustment"
```